yarn-project/pxe/README.md

```diff
@@ -1,10 +1,10 @@
-# AztecRPCServer & AztecRPCClient
+# PXEService & PXEClient
 
-`AztecRPCServer` is a server-side software that provides a set of apis for interacting with the Aztec network. It acts as a bridge between the network and client applications by exposing methods to manage accounts, deploy contracts, create transactions, and retrieve public and account-specific information. It provides a secure environment for the execution of sensitive operations, ensuring private information and decrypted data are not accessible to unauthorized applications. The AztecRPCServer is a critical component of the Aztec network, and its security and reliability are essential to the overall trustworthiness of user's data.
+`PXEService` is a server-side software that provides a set of apis for interacting with the Aztec network. It acts as a bridge between the network and client applications by exposing methods to manage accounts, deploy contracts, create transactions, and retrieve public and account-specific information. It provides a secure environment for the execution of sensitive operations, ensuring private information and decrypted data are not accessible to unauthorized applications. The PXEService is a critical component of the Aztec network, and its security and reliability are essential to the overall trustworthiness of user's data.
```

Comment:
maybe add:
 `PXEService` (pronounced pixie service) is a server-side....

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2552/commits/558c34ad4d2cd156302262fbef2ff0d82741e272 😉